src/Products/DateRangeInRangeIndex/exportimport.py

```diff
@@ -29,7 +30,7 @@ def _importNode(self, node):
             _.tagName: _.getAttribute('value')
             for _ in node.childNodes if _.nodeType == 1
         }
-        self.context.startindex = child_nodes["startindex"].encode('utf-8')
-        self.context.endindex = child_nodes["endindex"].encode('utf-8')
```

Comment:
Is this `encode` call no longer required for Python 2.7?

Reply:
I would be careful removing it here for Py 2.7. Better do this conditionally whether its already string/unicode (py27) or bytes/string (py3)?

Reply:
I don't think it was required there at all, because we are talking here about the key of the index not the content of the index. The test are fine and in my package I'll test it again with all versions, to make sure it works on Python 2.7 which is my main target for now anywhay.